### PR TITLE
Singlebuttonconfig

### PR DESCRIPTION
**Pull request by Peque**

### CODE REVIEW

**Comment by cua-cua** [comment]
Update the comment for ´static void after_moving(void)´. It is executed when the mouse stops, in general

**Comment by cua-cua** [comment]
Delete `set_target_goal function` from `set_run_sequence`

File: src/main.c [hunk 1]
```diff
@@ -103,69 +84,82 @@ static void after_moving(void)
...
+	save_maze();
 }
 
 /**
- * @brief Training routine.
+ * @brief Configure the robot and execute the search/run.
  */
-static void training(void)
+static void configure_and_solve(void)
 {
```
Comment (cua-cua) [inline src/main.c]: Update the name to `configure_and_move`

File: src/main.c [hunk 2]
```diff
@@ -103,69 +84,82 @@ static void after_moving(void)
 }
 
 /**
- * @brief Includes the functions to be executed during exploration phase.
+ * @brief Let the user configure the mouse forces before exploring or running.
```
[the remark this answers is not in the document]
Comment (Peque) [reply src/main.c]: fbfefc16092eaa37c2f718c29fa35e9509f121db

File: src/main.c (hunk 1 above)
Comment (Peque) [reply src/main.c]: 91cafc1eebca1275be5d81010a20b92c6e3963ff

**Pull request merged**